azure_mgmt_graph/lib/generated/azure_mgmt_graph/applications.rb

```diff
@@ -131,15 +131,38 @@ def create_async(parameters, custom_headers = nil)
     # Lists applications by filter parameters. Reference:
     # http://msdn.microsoft.com/en-us/library/azure/hh974476.aspx
     #
+    # @param api_version [String] Client Api Version.
```

Comment:
The issue seems unrelated to this PR, so we'll merge it and fix the issue separately.